### PR TITLE
Styling for bookstore app

### PR DESCRIPTION
**Pull request by chukwuemeka1234**
Hi there 👋 

- [ ] **In this PR, the styling for the bookstore app was completed according to the project requirement**

### CODE REVIEW

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @chukwuemeka1234, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [ ] For this project it is a requirement that you strictly follow the given design, including colors, fonts, layouts, margins, padding, etc. I will leave screenshots here so that you can compare and make the necessary adjustments.

- Given design: 
Screenshot from 2022-01-21 00-16-30

- Your design: 
Screenshot from 2022-02-05 03-11-06

- Highlited adjustments: 
Screenshot-from-2022-01-21-00-16-30

- [ ] kindly remove the scroll between the cards and ensure that books display cover the browser.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 1]
```diff
@@ -1,4 +1,338 @@
...
+  width: 5rem;
+}
+
+.edit {
+  padding-left: 1rem;
+  width: 3rem;
+  border: none;
+  cursor: pointer;
+  background-color: white;
+}
```
Comment (julie-ify) [inline src/index.css]: - [ ] Why don't you make this `Edit` have the same color as the `Delete` and `Comment` so as to match the design guideline and also as suggested by the previous reviewer?😄

Your Implementation | Zeplin Design
:---:|:---:
 |

File: src/index.css [hunk 2]
```diff
@@ -1,4 +1,338 @@
...
+.inputForm {
+  width: 72vw;
+  height: 4.813rem;
+  margin: 1.313rem 4.85rem 0;
+  padding: 0;
+  padding-bottom: 3rem;
+  border-top: solid 1px #b4b0b0;
+  background-color: #e7e3e3;
+  margin-left: 13%;
+}
```
Comment (julie-ify) [inline src/index.css:70]: - [ ] Why don't you align the `form` properly and make the form to be horizontal on the page so as to match the design guideline and also as suggested by the previous reviewer?😄

Your Implementation | Zeplin Design
:---:|:---:
 |

File: src/index.css [hunk 3]
```diff
@@ -1,4 +1,338 @@
...
+
+::-webkit-scrollbar {
+  display: none;
+}
+
+.user {
+  color: #0290ff;
+  font-size: 1.5rem;
+  padding-bottom: 1rem;
+}
```
Comment (julie-ify) [inline src/index.css:220]: - [ ] Why don't you align the `user` with the cards on the page so as to match the design guideline and also as suggested by the previous reviewer?😄

Your Implementation | Zeplin Design
:---:|:---:
 |

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @chukwuemeka1234,

Good job so far!

## Highlight!
- Good job implementing some of the suggested changes from the previous reviewer ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css (hunk 3 above)
Comment (chukwuemeka1234) [reply src/index.css:220]: The user is properly aligned on my own side.

File: src/index.css (hunk 2 above)
Comment (chukwuemeka1234) [reply src/index.css:70]: The form is well aligned on my own side.

File: src/index.css (hunk 3 above)
Comment (chukwuemeka1234) [reply src/index.css:220]: **I have one last review left. Please I have implemented all the requirements.**

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @chukwuemeka1234, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**